docs/Installation.md

```diff
@@ -1,8 +1,134 @@
+$ go mod tidy
+# or go mod tidy -compat=1.19
+# Copy the dependencies to vendor folder
+$ go mod vendor
+```
+
+4. By default the installation script will use the `redhat-appstudio-qe` GitHub organization for pushing changes to `infra-deployments` repository.
+
+**It is recommended to use your fork of [infra-deployments repo](https://github.com/redhat-appstudio/infra-deployments) in your GitHub org instead** - you can change the GitHub organization with environment variable `export MY_GITHUB_ORG=<name-of-your-github-org>`.
+
+<details>
+<summary><h4>Advanced installation guidelines</h4></summary>
+<h5>GitHub Application for Pipelines as Code<h5>
+<p>
+   Some tests could require you have a Github App created in order to test Component builds via Pipelines as Code.
+Such tests are [rhtap-demo](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/rhtap-demo/rhtap-demo.go) and [build](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/build/build.go).
+
+In this case, before you bootstrap a cluster, make sure you [created a Github App for your GitHub account](https://github.com/settings/apps). Fill in following details:
+</p>
+<ul><li> <b>GitHub App name</b>: unique app name</li> </ul>
+<ul><li> <b>Homepage URL</b>: some dummy URL (like https://example.com)</li></ul>
+<ul> <li> <b>Webhook</b>: mark as active and put some dummy URL to the <b>Webhook URL</b> field</li></ul>
+<ul> <li> <b>Permissions</b> and <b>Subscribe to events</b>: refer to <a href="https://pipelinesascode.com/docs/install/github_apps/#setup-manually">the guide in PaC documentation</a></li></ul>
+<ul> <li> <b>Where can this GitHub App be installed></b>: Any account</li> </ul>
+
+<p></p>
+
+Hit create, make a note of the <b>App ID</b> and navigate to the <b>Private keys</b> section where you generate a private key that gets downloaded automatically. Then export following environment variables (or if you're using .env file, update values of following variables):
+
+```bash
+export E2E_PAC_GITHUB_APP_ID='<YOUR_APP_ID>'
+
+export E2E_PAC_GITHUB_APP_PRIVATE_KEY=$(base64 < /PATH/TO/YOUR/DOWNLOADED/PRIVATE_KEY.pem)
+```
+
+<p>
+Navigate back to <a href="https://github.com/settings/apps">your GitHub App</a>, select Install App and select your GitHub org (the one that you're using in `MY_GITHUB_ORG` env var). Feel free to install it to all repositories of that organization or the forked repositories currently used by <a href="(https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/rhtap-demo/rhtap-demo.go)">rhtap-demo</a> and <a href="(https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/build/build.go">build tests</a>
+</p>
+
+</details>
+
+For bootstrapping a cluster, run the following command:
+
+   ```bash
+      make local/cluster/prepare
+   ```
+
+More information about how to deploy RHTAP
+are in the [infra-deployments](https://github.com/redhat-appstudio/infra-deployments) repository.
+
+### Building and running the e2e tests
```

Comment:
What be nice to enumerate all required/recommended env vars, and mention how to configure them